WalletWasabi/Helpers/EnvironmentHelpers.cs

```diff
@@ -90,45 +90,44 @@ public static string GetDataDir(string appName)
 			return directory;
 		}
 
-		public static string TryGetDefaultBitcoinCoreDataDir()
+		/// <summary>
+		/// Gets Bitcoin <c>datadir</c> parameter from:
+		/// <list type="bullet">
+		/// <item><c>APPDATA</c> environment variable on Windows, and</item>
+		/// <item><c>HOME</c> environment variable on other platforms.</item>
+		/// </list>
+		/// </summary>
+		/// <returns><c>datadir</c> or empty string.</returns>
+		/// <seealso href="https://en.bitcoin.it/wiki/Data_directory"/>
+		public static string GetDefaultBitcoinCoreDataDirOrEmptyString()
 		{
-			string directory = null;
+			string directory = "";
 
-			// https://en.bitcoin.it/wiki/Data_directory
-			try
+			if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
 			{
-				if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+				var localAppData = Environment.GetEnvironmentVariable("APPDATA");
+				if (!string.IsNullOrEmpty(localAppData))
 				{
-					var localAppData = Environment.GetEnvironmentVariable("APPDATA");
-					if (!string.IsNullOrEmpty(localAppData))
-					{
-						directory = Path.Combine(localAppData, "Bitcoin");
-					}
-					else
-					{
-						throw new DirectoryNotFoundException($"Could not find suitable default {Constants.BuiltinBitcoinNodeName} datadir.");
-					}
+					directory = Path.Combine(localAppData, "Bitcoin");
 				}
 				else
 				{
-					var home = Environment.GetEnvironmentVariable("HOME");
-					if (!string.IsNullOrEmpty(home))
-					{
-						directory = RuntimeInformation.IsOSPlatform(OSPlatform.OSX)
-							? Path.Combine(home, "Library", "Application Support", "Bitcoin")
-							: Path.Combine(home, ".bitcoin"); // Linux
-					}
-					else
-					{
-						throw new DirectoryNotFoundException($"Could not find suitable default {Constants.BuiltinBitcoinNodeName} datadir.");
-					}
+					Logger.LogDebug($"Could not find suitable default {Constants.BuiltinBitcoinNodeName} datadir.");
```

Comment:
Why not call `NotificationHelpers.Error(ex.ToUserFriendlyString());` before logging?

Reply:
I'm not sure whether `NotificationHelpers.Error()` is available at this point. Could you please test?

Reply:
You are right.